uint8 dataID
uint8 operationEnabled
uint8 actualMode
uint8 Endeffector_IsOpen
int32[6] actualPositions
int32[6] actualVelocities
int32[6] HomeOffset
int32[6] targetPositions
int32[6] targetVelocities
int32[6] targetAcceleration
uint8[6] PositionReached
uint8[2] dummy